.github/workflows/tests.yaml

```diff
@@ -0,0 +1,64 @@
+name: Run tests
+
+on:
+  # Run on any push
+  - push
+  # Run manually (from actions tab)
+  - workflow_dispatch
```

Comment:
what does this mean? does it mean we have to press a button to activate the tests in github?

Reply:
It just means we *can* push a button.